pipeline_dp/combiners.py

```diff
@@ -502,6 +504,9 @@ def create_compound_combiner(
             combiners.append(
                 SumCombiner(CombinerParams(budget_sum, aggregate_params)))
     if pipeline_dp.Metrics.PRIVACY_ID_COUNT in aggregate_params.metrics:
+        if aggregate_params.contribution_bounds_already_enforced:
```

Comment:
It's better to fail earlier, it look like here a better place is `AggregateParams` validation [here](https://github.com/OpenMined/PipelineDP/blob/3a7b87a5d943ea6b083c45934c8bb1110cbe5766/pipeline_dp/aggregate_params.py#L86)